source/_posts/en/tutorial/background.md

```diff
@@ -1,71 +1,93 @@
+
+Developers write custom business logic code to process the data from View. When the data from ViewModel changed, View will get updated automatically; When View declare the two-way bindings syntax, the framework will add the event handler to the DOM, and update the ViewModel's data when DOM get user's input.
+
+### MVVM diagram
 
 <img src="https://raw.githubusercontent.com/X-Jray/blog/master/assets/mvvm.png" width="540" alt="前端MVVM">
 
-如图所示，在前端 MVVM 框架中，往往没有清晰、独立的 Model 层。在实际业务开发中，我们通常按 **Web Component** 规范来组件化的开发应用，Model 层的域模型往往分散在在一个或几个 Component 的 ViewModel 层，而 ViewModel 层也会引入一些 View 层相关的中间状态，目的就是为了更好的为 View 层服务。
+In MVVM framework. There is no distinct boundary of which kinds of code belong to Model. In actual production development, we usually use **Web Component** standard to manage our code. The concept of Model sometimes been separated in one or many component's ViewModel, and ViewModel have some temporary state, to better serve the View layer/
 
-开发者在 View 层的视图模板中声明 **数据绑定**、 **事件绑定** 后，在 ViewModel 中进行业务逻辑的 **数据** 处理。事件触发后，ViewModel 中 **数据** 变更， View 层自动更新。因为 MVVM 框架的引入，开发者只需关注业务逻辑、完成数据抽象、聚焦数据，MVVM 的视图引擎会帮你搞定 View。因为数据驱动，一切变得更加简单。
+Developer write **declare bindings**, **event bind** in the View template, and write business logic process in ViewModel. After the event fired, the data of ViewModel changed and View updating automatically. With the power of MVVM framework, developers only need to focus on business logic, the abstraction of data and MVVM framework will help you make everything right in the View. Data-driven make everything simpler.
 
-MVVM框架的工作
+What MVVM framework do
 -----
+MVVM can increase the productivity of application development. It's amazing! But, what MVVM framework do?
 
-不可置否，MVVM 框架极大的提升了应用的开发效率。It's amazing！But，MVVM 框架到底做了什么？
-
-- 视图引擎
+- View Engine
 
-视图引擎：我是视图引擎，我为 View 层作为视图模板提供强力支持，开发者，你们不需要操作 DOM ，丢给我来做！
+View Engine: fully support for template parsing and rendering. As a developer, you don't need to known what is DOM and how to operate it.
 
-- 数据存取器
+- Data Store
 
-数据存取器：我是数据存取器，我可以通过 `Object.defineProperty()` API 轻松定义，或通过自行封装存取函数的方式曲线完成。我的内部往往封装了 **发布/订阅模式**，以此来完成对数据的监听、数据变更时通知更新。我是 **数据绑定** 实现的基础。
+Data store: implemented by `Object.defineProperty()` API, or completed by self-encapsulating functions. Integrated with pub/sub patten to watch the data changed and subscribe the view to update when changing the data. This is the basic of **data bindings**.
 
-- 组件机制
+- Component Mechanism
 
-组件机制：我是组件机制。有追求的开发者往往希望按照面向未来的组件标准 － **Web Components** 的方式开发，我是为了满足你的追求而生。MVVM 框架提供组件的定义、继承、生命周期、组件间通信机制，为开发者面向未来开发点亮明灯。
+Component Mechanism: Developer who have highly pursue hope to use the next generation of the component standard -- **Web Components**. San was born to satisfy your pursuit. The MVVM framework provides component definition, inheritance, lifecycle, and inter-component communication mechanisms to enable developers to develop bright lights for the future.
```

Comment:
Developer who have highly pursue hope to use the next generation of the component standard -> Cutting edge developers frequently ask for a development style aligned to the future proof standard - Web Component

Reply:
inter-component communication -> cross component communication